extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeGroup.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Wikia\PortableInfobox\Parser\Nodes;
+use Wikia\PortableInfobox\Parser\Parser;
+
+class NodeGroup extends Node {
```

Comment:
maybe we should add group empty logic here, that it requires at least one not empty node excluding header?